cross/python38/Makefile

```diff
@@ -86,8 +86,8 @@ python38_post_install: $(WORK_DIR)/python-cc.mk
 	@$(RUN) $(PYTHON_NATIVE) -m crossenv $(STAGING_INSTALL_PREFIX)/bin/python$(PKG_VERS_MAJOR_MINOR) $(WORK_DIR)/crossenv/
 	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) wget https://bootstrap.pypa.io/get-pip.py -O - | build-python
 	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) wget https://bootstrap.pypa.io/get-pip.py -O - | python
-	. $(WORK_DIR)/crossenv/bin/activate && $(RUN) build-pip install "setuptools==44.1.0"  "wheel==0.36.2" "cffi==1.14.5"
```

Comment:
If I were you I would really avoid using `>=` mechanisms. It would make the build non repeatables.
Similarly it would be worth updating all `cross/*` equivalents to the same versions to be on the safe side.

Reply:
Actually, this is superseeded by #4902 and should be removed from your PR.

Reply:
Yes, the `>=` was an ugly work around during the testing and the errors being thrown that's why I mentioned in first post.

Now 3.8.12 is in master I'll update and rebase instead of using the combination of clone of th0ma7 3.8.12 branch from his repo along with my sickchill copied to that clone.